src/components/gcds-fieldset/gcds-fieldset.tsx

```diff
@@ -0,0 +1,105 @@
+import { Component, Prop, Element, Host, Watch, h } from '@stencil/core';
+import { assignLanguage } from '../../utils/utils';
+
+@Component({
+  tag: 'gcds-fieldset',
+  styleUrl: 'gcds-fieldset.css',
+  shadow: true,
+})
+export class GcdsFieldset {
+
+  private lang: string;
+
+  @Element() el: HTMLElement;
+
+  /**
+   * The unique identifier for the component
+   */
+  @Prop({ reflect: true, mutable: false }) fieldsetId!: string;
+  /**
+   * The title for the contents of the fieldset
+   */
+  @Prop({ reflect: true, mutable: false }) legend!: string;
+
+  /**
+   * Flag the contents are required
+   */
+  @Prop({ reflect: true, mutable: false }) required: boolean;
+  /**
+   * Error message for an invalid input element.
+   */
+  @Prop({ reflect: true, mutable: true }) errorMessage: string;
+
+  @Watch('errorMessage')
+  validateErrorFieldset() {
+    if (this.disabled) {
+      this.errorMessage = "";
+    }
+  }
+
+  /**
+    * Hint displayed below the label.
```

Comment:
Maybe use title instead of label to be consistent with the description for legend?